.github/workflows/ci_linux_x64_clang_debug.yml

```diff
@@ -39,23 +39,32 @@ jobs:
       - environment=${{ needs.setup.outputs.runner-env }}
       - cpu
       - os-family=Linux
+    container: gcr.io/iree-oss/base@sha256:dc314b4fe30fc1315742512891357bffed4d1b62ffcb46258b1e0761c737b446
+    defaults:
+      run:
+        shell: bash
     env:
       BUILD_DIR: build-debug
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@v4.1.7
         with:
           submodules: true
-      - name: "Building IREE in Debug configuration"
-        env:
-          IREE_WRITE_REMOTE_CCACHE: ${{ needs.setup.outputs.write-caches }}
+      - name: Install Python requirements
+        run: pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
+      # Note: not using ccache here. Debug builds need a lot of cache space
+      # and caching only provides marginal build time improvements.
```

Comment:
Here's a 2GB cache getting 11 / 5262 cache hits:
https://github.com/iree-org/iree/actions/runs/10411642302/job/28835942071#step:15:6

Soooo not possible using these cache settings and a local cache (backed up GitHub Actions Cache for save/restore). Might be able to use different settings like sccache or more compression, but it seems like the cache only saves 2 minutes on an 11 minute build for `-DCMAKE_BUILD_TYPE=Debug` anyways 🤷‍♂️